code/game/gamemodes/cult/ritual.dm

```diff
@@ -51,34 +50,34 @@
 	. = ..()
 
 /obj/item/melee/cultblade/dagger/attack_self(mob/user)
-	if(!iscultist(user))
+	if(!IS_CULTIST(user))
 		to_chat(user, "<span class='warning'>[src] is covered in unintelligible shapes and markings.</span>")
 		return
 	scribe_rune(user)
 
 /obj/item/melee/cultblade/dagger/proc/narsie_rune_check(mob/living/user, area/A)
 	var/datum/game_mode/gamemode = SSticker.mode
 
-	if(gamemode.cult_objs.cult_status < NARSIE_NEEDS_SUMMONING)
-		to_chat(user, "<span class='cultitalic'><b>[SSticker.cultdat.entity_name]</b> is not ready to be summoned yet!</span>")
+	if(gamemode.cult_team.cult_status < NARSIE_NEEDS_SUMMONING)
+		to_chat(user, "<span class='cultitalic'><b>[GET_CULT_DATA(entity_name, "Your god")]</b> is not ready to be summoned yet!</span>")
```

Comment:
```suggestion
		to_chat(user, "<span class='cultitalic'><b>[GET_CULT_DATA(entity_name, "your god")]</b> is not ready to be summoned yet!</span>")
```
And this is why I'd strongly suggest defines

Reply:
This starts the sentence, it needs to be capitalized